### ISSUE TITLE
feat: support multi line input

### ISSUE THREAD

**Issue by mirsella** [console-rs/dialoguer#297]
Hello !

Would be very useful to have a way of writing multiple lines in an Input.
For example, copying and pasting this text: 
```
foo
bar
```
Would take only foo.

To know when to stop, maybe we can stop when getting an empty string back, for example with `std::io::stdin().read_line(&mut buffer)` will return 0 (because it didn't read anything) and not modify buffer.

**Comment by ThatFrogDev** [comment]
Yes, this would be indeed very awesome! I need that for my app too. Thanks for the request!
